wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/wdlom2wom/graph/WorkflowGraphElementToGraphNode.scala

```diff
@@ -48,6 +56,16 @@ object WorkflowGraphElementToGraphNode {
       val callNodeMakerInputs = CallNodeMakerInputs(ce, a.linkableValues, a.linkablePorts, a.availableTypeAliases, a.workflowName, a.insideAScatter, a.callables)
       CallElementToGraphNode.convert(callNodeMakerInputs)
   }
+
+  def validateAssignmentType(womExpr: WomExpression, womType: WomType): ErrorOr[Unit] = {
```

Comment:
[ToL] I was going to say "just use a boolean?" but the way you can map over it is actually pretty brilliant

Reply:
😎